### PR TITLE
Search view client

### PR DESCRIPTION
**Pull request by vkatsuba**

### CODE REVIEW

File: apps/example_site/priv/lib/js/search-view.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+jQuery( document ).ready(function() {
+  jQuery( 'input.form-control' ).on('input', function() {
```
Comment (mworrell) [inline apps/example_site/priv/lib/js/search-view.js:2]: Should we make this specific for `name=qs` as otherwise we might add this to other form controls.

Comment (vkatsuba) [reply apps/example_site/priv/lib/js/search-view.js:2]: Yes, this is will be better, will change, thanks.

Comment (vkatsuba) [reply apps/example_site/priv/lib/js/search-view.js]: Done.

**Comment by vkatsuba** [comment]
Implemented in the scope of https://github.com/zotonic/zotonic_example/pull/7.

**Pull request closed without merging**